### ISSUE TITLE
[FEATURE] SilentSneak

### ISSUE THREAD

**Issue by liquidsquid1** [CCBlueX/LiquidBounce#2506]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

This module should make it so sneaking does **not** make an appearance on your screen, f5 or not. This would make it easier to play with sneak based scaffolds.

**Comment by SenkJu** [comment]
Already in the client. Set sneak to `Vanilla`.

![image](https://github.com/CCBlueX/LiquidBounce/assets/18741573/e80c8a60-ba49-4d44-b532-80f8586f11a0)

**Comment by liquidsquid1** [comment]
I believe you have misunderstood my point. I know that there is a sneak mode vanilla but the “SilentSneak” would instead hide the player sneaking instead of forcing the player to be in a permanent sneak state. This means that your camera would not move up and down when scaffolding, etc.

**Comment by EclipsesDev** [comment]
> I believe you have misunderstood my point. I know that there is a sneak mode vanilla but the “SilentSneak” would instead hide the player sneaking instead of forcing the player to be in a permanent sneak state. This means that your camera would not move up and down when scaffolding, etc.

ahh u mean packet based sneak? like server-sided only sneak right?

**Comment by EclipsesDev** [comment]
meaning sneaking on server only (not client-sided)

**Comment by liquidsquid1** [comment]
> meaning sneaking on server only (not client-sided)

Yeah, like holding shift doesn’t apply the camera movement, meaning it can integrate with other modules like Scaffold (Eagle)
